migrate/v0_17/testdata/genesis-v16.json

```diff
@@ -1,2280 +1,2725 @@
 {
-  "genesis_time": "2022-01-19T16:00:00Z",
```

Comment:
The reformatting noise in this diff for these two files is the result of running vscode "Prettify JSON", would be happy to update my vscode config to match what the rest of the team is using!

Reply:
We use the `jq` default of two spaces

Reply:
Can pipe through jq to fix formatting also